README.md

```diff
@@ -70,6 +70,12 @@ To add users:
 
 To enable stubbing of the ```PROXY_PAYMENTS``` set the ```PROXY_PAYMENTS_STUB``` environment variable to the desired url.
 
+
+Add entry to /etc/hosts
+```
+127.0.0.1   ccd-api-gateway
```

Comment:
api gateway must be accessible form within docker container (visible as ccd-api-gateway) and from web browser (visible as localhost:3453), but xui have one config value for it. The workaround is to add alias ccd-api-gateway->localhost in /etc/hosts

Reply:
@tomaszpowroznik, do you mean this should be added to our computers (host) or to the xui pod?
I think you mean the hosts. If so, could we not add this to the xui pod instead?

Reply:
it is about local machine and local environment only